### ISSUE TITLE
Permission denied when trying to enable mods via arkmanager

### ISSUE THREAD

**Issue by realdegrees** [Hermsi1337/docker-ark-server#11]
I already kind of fixed it but still wanted to let you know.
I setup a server by running the image with basic settings, edited the inis, ran the server and everything was fine.
Then I installed S+ and wanted to enable it but the user account doesn't seem to have permissions for some reason:
```
docker exec -it ark_server bash
steam@83e0782bda2a:/app$ whoami
steam
steam@83e0782bda2a:/app$ which arkmanager
/usr/local/bin/arkmanager
steam@83e0782bda2a:/app$ arkmanager enablemod 895711211
Running command 'enablemod' for instance 'main'
/usr/local/bin/arkmanager: line 2938: /etc/arkmanager/instances/main.cfg: Permission denied
```

This occured with other mods as well. HOWEVER, after disabling the mod the error message went away.

```
steam@83e0782bda2a:/app$ arkmanager enablemod 895711211
Running command 'enablemod' for instance 'main'
/usr/local/bin/arkmanager: line 2938: /etc/arkmanager/instances/main.cfg: Permission denied
steam@83e0782bda2a:/app$ arkmanager disablemod 895711211
Running command 'disablemod' for instance 'main'
steam@83e0782bda2a:/app$ arkmanager enablemod 895711211
Running command 'enablemod' for instance 'main'
```

**Comment by Hermsi1337** [comment]
Sorry for my very delayed response.. 

The config `/etc/arkmanager/instances/main.cfg` is intended as a "global" config which should be overwritten by the user-specific config.  
It's also not a volume, so changes are lost when restarting the container.

I'll have to figure out, how config overwriting works in `arkmanager` and how to save changes only to the user-specific config.

**Comment by Hermsi1337** [comment]
This is fixed by #25 .

The mods are now automagically installed when passed via env `GAME_MOD_IDS`.

Also, you now have access to the `arkmanager`-config in `${SERVER_VOLUME}/arkmanager`
